### PR TITLE
Feature: Playbook base64 encoded values

### PR DESCRIPTION
**Pull request by pavangudiwada**
Users can use base64 encoded keys with **external playbooks repos** and **git change audit** action.

### CODE REVIEW

File: playbooks/robusta_playbooks/git_change_audit.py [hunk 1]
```diff
@@ -64,38 +76,42 @@ def git_change_audit(event: KubernetesAnyChangeEvent, action_params: GitAuditPar
...
+            if obj_diff(event.obj.spec, old_spec, action_params.ignored_changes):  # we have a change in the spec
+                git_repo.commit_push(
+                    hikaru.get_yaml(event.obj.spec),
+                    path,
+                    name,
+                    f"Update {event.obj.kind} named {event.obj.metadata.name} on namespace {namespace}",
+                    action_params.cluster_name,
+                )
+    except Exception as e:
+        logging.exception(e)
```
Comment (arikalon1) [inline playbooks/robusta_playbooks/git_change_audit.py:116]: Don't think we need that
The exc_info logs the stack trace

File: src/robusta/utils/base64_utils.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import base64
+import logging
+
+
+def is_base64_encoded(value: str) -> bool:
+    try:
+        if base64.b64encode(base64.b64decode(value)) == bytes(value, 'utf-8'):  # key is b64
```
Comment (arikalon1) [inline src/robusta/utils/base64_utils.py:7]: I think you can just do:
`return base64.b64encode(base64.b64decode(value)) == bytes(value, 'utf-8')`  # value is b64

File: src/robusta/utils/base64_utils.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+            return True
+        else:
+            return False
+    except Exception as e:
+        logging.error("Error during base64 check")
```
Comment (arikalon1) [inline src/robusta/utils/base64_utils.py:12]: I don't think we need to print anything here
It's not a real error
When we get a value which isn't b64, we expect that to happen. It's just a way to test if value is b64 encoded
Printing this as errors will just spam the logs

**Review by arikalon1 — APPROVED** [review]
Nice work!

Left some minor comments
Please add the relevant information to the docs, on how to use the b64 encoded values, and also how to create it, from the multi line keys.

File: src/robusta/utils/base64_utils.py (hunk 2 above)
Comment (pavangudiwada) [reply src/robusta/utils/base64_utils.py:7]: Understood, fixed.

File: src/robusta/utils/base64_utils.py (hunk 3 above)
Comment (pavangudiwada) [reply src/robusta/utils/base64_utils.py:12]: Fixed

File: playbooks/robusta_playbooks/git_change_audit.py (hunk 1 above)
Comment (pavangudiwada) [reply playbooks/robusta_playbooks/git_change_audit.py:116]: Got it.

**Pull request merged**